src/viewer/viewer.ts

```diff
@@ -40,8 +40,11 @@ import { testTextureSupport } from './gl-utils'
 
 import Buffer from '../buffer/buffer'
 
-const pixelBufferFloat = new Float32Array(4)
-const pixelBufferUint = new Uint8Array(4)
+const pixelBufferFloat = new Float32Array(4 * 25)
+const pixelBufferUint = new Uint8Array(4 * 25)
+
+const pixelOrder = [12,7,13,17,11,6,8,18,16,2,14,22,10,1,3,9,19,23,21,15,5,0,4,24,20]
```

Comment:
Could you add a comment what the order mean geometrically? Thanks.

Reply:
Will do (It starts at the centre and looks at pixels by distance to centre)

Reply:
Done - are we good to merge?